solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -1115,8 +1027,158 @@ describe("audius-data", function () {
     });
   });
 
+  it("delegate creates a track (manage entity) + all validation errors", async function () {
+    // create user and delegate
+    const delegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair: adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    // create track with delegate
+    const trackMetadata = randomCID();
+    const trackID = randomId();
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program,
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: SystemProgram.programId, // missing PDA
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program,
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA: SystemProgram.programId, // missing PDA
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
```

Comment:
again, these should be broken up into separate test cases for clarity.

Reply:
let's not block on this, test coverage is more important / more cases actually is less readable imo